config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_home_visit.danger_signs.breaking_water = Breaking of water
+report.pregnancy_home_visit.danger_signs.easily_tired = Getting tired easily
+report.pregnancy_home_visit.danger_signs.face_hand_swelling = Swelling of face and hands
+report.pregnancy_home_visit.danger_signs.breathlessness = Breathlessness
+report.pregnancy_home_visit.safe_pregnancy_practices = Safe Pregnancy Practices
+report.pregnancy_home_visit.safe_pregnancy_practices.malaria = Malaria
+report.pregnancy_home_visit.safe_pregnancy_practices.malaria.llin_use = Patient uses a long-lasting insecticidal net (LLIN)
+report.pregnancy_home_visit.safe_pregnancy_practices.iron_folate = Iron Folate
+report.pregnancy_home_visit.safe_pregnancy_practices.iron_folate.iron_folate_daily = Taking iron folate daily 
+report.pregnancy_home_visit.safe_pregnancy_practices.deworming = Deworming
+report.pregnancy_home_visit.safe_pregnancy_practices.deworming.deworming_med = Received deworming medication
+
+report.pregnancy_home_visit.safe_pregnancy_practices.hiv_status = HIV Status
+report.pregnancy_home_visit.safe_pregnancy_practices.hiv_status.hiv_tested = Tested for HIV in the past 3 months.
+report.pregnancy_home_visit.safe_pregnancy_practices.tetanus = Tetanus
+report.pregnancy_home_visit.safe_pregnancy_practices.tetanus.tt_imm_received = Received Tetanus Toxoid (TT) immunizations during pregnancy
+report.pregnancy_home_visit.safe_pregnancy_practices.safe_pregnancy_practices = Safe pregnancy practices 
+
+report.pregnancy.gestational_age = Gestational Age
+report.pregnancy.gestational_age.register_method = Register Method
+report.pregnancy.gestational_age.register_method.register_note =  
+report.pregnancy.gestational_age.register_method.lmp_method = Lmp Method
+
+report.pregnancy.gestational_age.method_lmp = LMP
+report.pregnancy.gestational_age.method_lmp.u_lmp_date =  
+report.pregnancy.gestational_age.method_lmp.method_lmp = 
+report.pregnancy.gestational_age.method_approx = LMP Method Approx
+report.pregnancy.gestational_age.method_approx.lmp_approx = Report the pregnancy in weeks or months
+report.pregnancy.gestational_age.method_approx.lmp_approx_weeks = Weeks
+report.pregnancy.gestational_age.method_approx.lmp_approx_months = Months
+report.pregnancy.gestational_age.method_edd = EDD
+report.pregnancy.gestational_age.method_edd.u_edd = Date of Delivery
+
+report.pregnancy.gestational_age.method_lmp_summary = LMP Summarry
+
+report.pregnancy.gestational_age.g_lmp_date_8601 = LMP 8601
+report.pregnancy.gestational_age.g_lmp_date = LMP Date
+report.pregnancy.gestational_age.g_edd_8601 = EDD 8601
+report.pregnancy.gestational_age.g_edd = EDD
+report.pregnancy.gestational_age.method_none = Method None
+report.pregnancy.gestational_age.method_none.no_info_pregnancy_reason = Reason for registering patients
+
+report.pregnancy.anc_visits_hf = ANC Visits to Facility
+report.pregnancy.anc_visits_hf.anc_visits_hf_past = ANC Visits at Health Facility (Past)
```

Should this also be sentence case?

```suggestion
report.pregnancy.anc_visits_hf.anc_visits_hf_past = Past ANC Health Facility Visits
```